source/ui/Application.ooc

```diff
@@ -15,4 +15,5 @@ Application: class {
 		this _arguments free()
 		super()
 	}
+	processEvents: virtual func
```

Comment:
It is here because in the future it could be possible to process other types of events even when gui is off (like Timer events)